addons/controllers/certificates_test.go

```diff
@@ -0,0 +1,90 @@
+// Copyright 2022 VMware, Inc. All Rights Reserved.
```

Comment:
Why do we have both `webhooks/certificates_test.go` and `controllers/certificates_test.go`?

Reply:
controllers/certificates_test.go are the integration tests. they required envtest. The ones under webhooks/certificates.test.go are the unit test. They do not requried envtest. 
The rest of the work (next patches will probably have their code live under controllers/ as they only apply to the addons package.